docs/assisting-with-translations.md

```diff
@@ -68,3 +65,90 @@ translations at once:
+* We are happy to help in case you need assistance.
+
+### 5.	Choose what you want to translate
+* Click on "Dashboard" on the top.
+* Once you are on the "Dashboard", click on "Languages" on the left side and select your language.
+* You will see a lot of different resources and their progress.
+* Each resource consists of a number of strings, with a string being for example a paragraph or headline.
+* Each string has three possible states: "untranslated", "translated but unreviewed", and "reviewed". Only the first state "untranslated" is relevant for new volunteers. However, if you find a "translated but unreviewed" string that contains obvious mistakes, you are free to correct them. "Reviewed" strings can only be changed or unreviewed by reviewers.
+* **The first resource "bitcoin.org" contains all strings of the main page. If you are no Bitcoin expert, start here.**
+* **Everything else that follows starts with "devdocs...", indicating that these files are part of the developer documentation. It is recommended that you only try to translate the developer documentation if you are an experienced Bitcoin user and/or developer with a profound understanding.**
+
+### 6.	Start translating
+* You must be a native speaker for the language you choose to translate.
+* Please be careful to preserve the original meaning of each text.
+* Sentences and popular expressions should sound native in your language.
+* Translations need to be reviewed by a reviewer or coordinator before
+  publication.
+* Once reviewed, coordinators will notify the Team Leaders that a certain translation is ready for publication.
+* **In doubt, please contact coordinators on Transifex. That'll be much
+  appreciated.**
+  
+### 7.	Take a look at the Responsibilities and Tasks for each User Role
+* The next section [Responsibilities and Tasks for each User Role](#responsibilities-and-tasks-for-each-user-role) outlines - as the title says - the responsibilities and tasks for each user role.
+
+___
+
+## Responsibilities and Tasks for each User Role
+
+This section shall outline the responsibilities and tasks for each user role.
+
+### 1. Team Leaders
+Team Leaders are currently Simon AKA “Komodorpudel” (Telegram: “Komodorpudel”) and Apple AKA “ajsolbrilla”.
+
+* Oversight on the complete translation efforts on Transifex.
+* Keeping track on everything.
+* Being a contact person for all sorts of questions that cannot be answered by language coordinators.
+* Promoting or demoting users (e.g. promoting a reviewer to coordinator).
+* Managing groups that have no active coordinator.
+
+### 2.	Coordinators
+Various people across all language teams are coordinators. For a number of languages, no active coordinator exists. If there are any questions or you want to assist by becoming a coordinator, write one of the Team Leaders.
+* If man power is needed: Translating, thereby using translations provided by the glossary if applicable and striving for consistency across strings.
```

Comment:
Please change:

If man power is needed:

to:

When additional help is needed: